docs/api/java/getting-started.md

```diff
@@ -0,0 +1,831 @@
+# Getting started
+
+1. Connect to the zksync network.
+2. Deposit assets from Ethereum into zksync.
+3. Check Balance 
+4. Transfer funds (ERC20 tokens) into the zksync network
+5. Withdraw funds (Native coins)
+6. Deploy contract with create method
+
+## Prerequisite
+This guide assumes that you are familiar with the basics of java and you have java configured on your local machine.
```

Comment:
"you are familiar with the basics of Java", including a link to the Java documentation. Use capitals